src/main/java/com/minecolonies/inventory/InventoryCitizen.java

```diff
@@ -550,6 +546,15 @@ public ItemStack decrStackSize(int index, int count)
         {
             return null;
         }
+    }    /**
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ATrailingCommentCheck)